Description: drop the building of the media extension of python to avoid the
 dependency on libgtk-media
Author: Fabrice Coutadeur <coutadeurf@gmail.com>
Last-Updated: 2013-09-10

--- a/wxPython/setup.py
+++ b/wxPython/setup.py
@@ -423,24 +423,6 @@
                 )
 wxpExtensions.append(ext)
 
-
-mediaLibs = libs[:]
-if not MONOLITHIC and findLib('media', libdirs):
-    mediaLibs += makeLibName('media')
-swig_sources = run_swig(['media.i'], 'src', GENDIR, PKGDIR,
-                        USE_SWIG, swig_force, swig_args, swig_deps)
-ext = Extension('_media', swig_sources,
-                include_dirs =  includes,
-                define_macros = defines,
-                library_dirs = libdirs,
-                libraries = mediaLibs,
-                extra_compile_args = cflags,
-                extra_link_args = lflags,
-                **depends
-                )
-wxpExtensions.append(ext)
-
-
 swig_sources = run_swig(['webkit.i'], 'src', GENDIR, PKGDIR,
                         USE_SWIG, swig_force, swig_args, swig_deps)
 ext = Extension('_webkit', swig_sources,
